src/test/operators/print_test.cpp

```diff
@@ -311,13 +311,6 @@ TEST_F(OperatorsPrintTest, SegmentType) {
 
 TEST_F(OperatorsPrintTest, EmptyTable) {
   auto tab = Hyrise::get().storage_manager.get_table(_table_name);
-  Segments empty_segments;
-  auto column_1 = std::make_shared<opossum::ValueSegment<int>>();
-  auto column_2 = std::make_shared<opossum::ValueSegment<pmr_string>>();
-  empty_segments.push_back(column_1);
-  empty_segments.push_back(column_2);
-  tab->append_chunk(empty_segments);
```

Comment:
If the rules for empty chunks shall be tightened, adding two consecutive empty segments should not be allowed, right? Do we expect a failure in this scenario? If so, how about adding an `EXPECT_THROW` test case here?
Currently, this scenario doesn't cause a failure. Adding an assertion in `Table::append_chunk` as it was done in the table constructor could solve this issue.